src/themes/move-theme/typography.js

```diff
@@ -15,29 +15,34 @@ export const fontTokens: FontTokensT = {
 const secondaryFontFamily =
   'UberMove, UberMoveText, system-ui, "Helvetica Neue", Helvetica, Arial, sans-serif';
 
+const monoFontFamily = 'UberMoveMono, "Lucida Console", Monaco, monospace';
```

Comment:
Does design have preferences for backup fonts?

Reply:
They approved this while working on the data-table, but I can bring it up specifically